### PR TITLE
License

### PR DESCRIPTION
**Pull request by brianteeman**
Make sure that we correctly refer to LICENSE.txt and GPL 2 or later

(does not include vendor files in /libraries/

### CODE REVIEW

File: modules/mod_breadcrumbs/tmpl/default.php [hunk 1]
```diff
@@ -30,23 +29,24 @@
...
 		// Make a link if not the last item in the breadcrumbs
 		$show_last = $params->get('showLast', 1);
-
 		// Generate the trail
 		foreach ($list as $key => $item) :
 			if ($key !== $last_item_key) :
 				// Render all but last item - along with separator ?>
 				<li itemprop="itemListElement" itemscope itemtype="https://schema.org/ListItem">
 					<?php if (!empty($item->link)) : ?>
-						<a itemprop="item" href="<?php echo $item->link; ?>" class="pathway"><span itemprop="name"><?php echo $item->name; ?></span></a>
```
Comment (HLeithner) [inline modules/mod_breadcrumbs/tmpl/default.php]: I think you mixed a wrong branch in this pr.

File: language/en-GB/en-GB.mod_finder.ini [hunk 2]
```diff
@@ -38,6 +38,7 @@ MOD_FINDER_FIELD_OPENSEARCH_DESCRIPTION="Some browsers can add support for your
 MOD_FINDER_FIELD_OPENSEARCH_LABEL="OpenSearch Autodiscovery"
 MOD_FINDER_FIELD_OPENSEARCH_TEXT_DESCRIPTION="Text displayed in supported browsers when adding your site as a search provider."
 MOD_FINDER_FIELD_OPENSEARCH_TEXT_LABEL="OpenSearch title"
+MOD_FINDER_OPENSEARCH_NAME="OpenSearch"
```
Comment (HLeithner) [inline language/en-GB/en-GB.mod_finder.ini]: Unrelated change

File: templates/protostar/index.php [hunk 3]
```diff
@@ -88,7 +88,7 @@
 }
 
 // Check for a custom CSS file
-JHtml::_('stylesheet', 'user.css', array('version' => 'auto', 'relative' => true));
+JHtml::_('stylesheet', 'z.css', array('version' => 'auto', 'relative' => true));
```
Comment (HLeithner) [inline templates/protostar/index.php]: Unrelated change

**Comment by alikon** [comment]
I have tested this item :white_check_mark: successfully on aff4208fb1398c9ec566321c92b3d86ab411fb92<br /><br />review<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/26342">issues.joomla.org/tracker/joomla-cms/26342</a>.</sub>

**Comment by richard67** [comment]
I have tested this item :white_check_mark: successfully on aff4208fb1398c9ec566321c92b3d86ab411fb92<br /><br />Reviewed all changes.<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/26342">issues.joomla.org/tracker/joomla-cms/26342</a>.</sub>

**Comment by alikon** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/26342">issues.joomla.org/tracker/joomla-cms/26342</a>.</sub>

File: language/en-GB/en-GB.mod_finder.ini [hunk 4]
```diff
@@ -1,6 +1,6 @@
 ; Joomla! Project
 ; Copyright (C) 2005 - 2019 Open Source Matters. All rights reserved.
-; License GNU General Public License version 2 or later; see LICENSE.txt, see LICENSE.php
+; License GNU General Public License version 2 or later; see LICENSE.txt, see LICENSE.txt
```
Comment (Quy) [inline language/en-GB/en-GB.mod_finder.ini]:
```suggestion
; License GNU General Public License version 2 or later; see LICENSE.txt
```

Comment (brianteeman) [reply language/en-GB/en-GB.mod_finder.ini]: that was the one file I megred incorrectly. thanks

**Comment by richard67** [comment]
I have tested this item :white_check_mark: successfully on 0d2b5046734655dd4ef1dcec29623e4b3dfea853<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/26342">issues.joomla.org/tracker/joomla-cms/26342</a>.</sub>

**Comment by viocassel** [comment]
I have tested this item :white_check_mark: successfully on 0d2b5046734655dd4ef1dcec29623e4b3dfea853<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/26342">issues.joomla.org/tracker/joomla-cms/26342</a>.</sub>

**Comment by richard67** [comment]
I have tested this item :white_check_mark: successfully on ba64c90938ef367b0e7fb95617964e3f0824a035<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/26342">issues.joomla.org/tracker/joomla-cms/26342</a>.</sub>

**Comment by HLeithner** [comment]
Thanks for the work to get the Joomla license clean, not only in this PR.

**Comment by brianteeman** [comment]
Thanks
